.editorconfig

```diff
@@ -571,6 +571,17 @@ MA0053.exceptions_should_be_sealed = true
 # Also report diagnostic for types that define (new) virtual members.
 MA0053.class_with_virtual_member_shoud_be_sealed = true
 
+# MA0076: Do not use implicit culture-sensitive ToString in interpolated strings
+#
+# Interpolated strings have performance benefits.
+dotnet_diagnostic.MA0076.severity = none
+
+# MA0089: Optimize string method usage
+#
+# Duplicate of CA1865-CA1867 /
+# For string.Join, results in verbose code for very little gain.
```

Comment:
Same.

Reply:
To comply with this, every "offer" message in `KeyExchange` needs to be duplicated or otherwise #if-def-ed to use `string.Join(',', ..` or `string.Join(",", ..`. But the methods perform the same:

```c#
        private static readonly string[] s_ciphers = ["aes128-ctr", "aes192-ctr", "aes256-ctr", "aes128-cbc", "aes192-cbc", "aes256-cbc", "3des-cbc", "blowfish-cbc", "twofish-cbc", "twofish192-cbc", "twofish128-cbc", "twofish256-cbc", "arcfour", "arcfour128", "arcfour256", "cast128-cbc"];

        [Benchmark]
        public string JoinString() => string.Join(",", s_ciphers);

        [Benchmark]
        public string JoinChar() => string.Join(',', s_ciphers);
```

| Method     | Mean     | Error    | StdDev   | Gen0   | Allocated |
|----------- |---------:|---------:|---------:|-------:|----------:|
| JoinString | 79.95 ns | 0.686 ns | 0.642 ns | 0.1912 |     400 B |
| JoinChar   | 80.61 ns | 0.901 ns | 0.843 ns | 0.1912 |     400 B |